webapp/src/components/dot_menu/dropdown.tsx

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {Strategy} from '@floating-ui/react';
```

Comment:
I assume this is the new "proper" way of doing things?

Reply:
Yep